docs/build/develop-distributed-algos.md

```diff
@@ -48,7 +48,7 @@ modules:
 
 ```
 
-Which drives the configuration of the modules comprising the service; in this case, it's just the *hello_world* module, which itself is defined in the services/modules/hello_world/module.yaml:
+that drives the configuration of the modules comprising the service; in this case, it's just the `hello_world` module, which itself is defined in the `services/modules/hello_world/module.yaml`:
```

Comment:
Here and below, 'that' introduces a restrictive clause. Thus, 'which' would be more correct, since `services.yaml` is the only file with this name in the `services` directory.

Reply:
Got it.